src/components/Routes/VirtualAssistant.tsx

```diff
@@ -1,14 +1,20 @@
 import React, { Fragment } from 'react';
 import { Route, Routes } from 'react-router-dom';
-import { ScalprumComponent } from '@scalprum/react-core';
+import { useAtom } from 'jotai';
+import { ScalprumComponent, ScalprumComponentProps } from '@scalprum/react-core';
 import { useFlags } from '@unleash/proxy-client-react';
 
+import { virtualAssistantOpenAtom, virtualAssistantStartInputAtom } from '../../state/atoms/virtualAssistantAtom';
+
 import './virtual-assistant.scss';
 
 const flaggedRoutes: { [flagName: string]: string } = { 'platform.va.openshift.insights': '/openshift/insights/*' };
 
 const VirtualAssistant = () => {
-  const viableRoutes = ['/', '/insights/*', '/settings/*', '/subscriptions/overview/*', '/subscriptions/inventory/*', '/subscriptions/usage/*'];
+  const [isOpen, setOpen] = useAtom(virtualAssistantOpenAtom);
+  const [startInput, setStartInput] = useAtom(virtualAssistantStartInputAtom);
+  // TODO: If a route that results in a 404 is not in this list, the Virtual Assistant will not be displayed. :(
```

Comment:
I'm not sure about the best way to get around the viableRoutes on 404 (I added the wildcard for testing)

Add the isHidden var to jotai and check it here? https://github.com/RedHatInsights/insights-chrome/blob/master/src/components/ChromeRoute/ChromeRoute.tsx#L35

Reply:
Met with Justin offline. Due to the nature of the VA (and it being a federated module) - we can simply render a scalprum component for VA on the 404 page (and in other locations) as the URL will not be predictable.

There is one issue we will need to solve however around state management (so on pages where the VA is already present and open we do not reset the chat history/reopen it). We can manage that state inside the VA itself like we have done for the notifications panel singleton instance.